com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/State.java

```diff
@@ -34,8 +34,9 @@
  */
 public class State implements Serializable {
 
-    private static final long serialVersionUID = -275410118302470803L;
+    private static final long serialVersionUID = -275410118302470802L;
```

Comment:
Old version is incompatible with the new one.
Changing version will trown an exception and new state will be created.